packages/driver/src/cypress/stack_utils.js

```diff
@@ -216,14 +220,23 @@ const getSourceDetailsForLine = (projectRoot, line) => {
 
   const sourceDetails = getSourceDetails(generatedDetails)
   const originalFile = sourceDetails.file
+
+  if (!originalFile) {
+    // this is an edge case: could not parse the stack trace
+    // let's not warn about it, but at least allow the user
+    // to see the original error by being defensive about it
+    /* eslint-disable-next-line no-console */
+    console.warn(`Could not get the original source file from line "${line}"`)
```

Comment:
I don't think we need to output this to the console. There's nothing the user can do about it, so it's just adding noise to the console output.

Reply:
it is so much easier for the user to report when complaining that their stack trace is missing :) But sure, I can remove it. By the way, we never use `debug` clientside really

Reply:
Their stack trace won't be missing. The only consequence is that any lines we can't get the source file for won't be open-able by IDE.

Reply:
updated